src/components/about/About.tsx

```diff
@@ -1,5 +1,6 @@
 import aboutImg from '@/assets/about.jpg';
 
+import { FooterComponent } from '../footer/Footer.component';
```

Comment:
Naming: when u creates variable or functions or other data structures u don't call them like:
```ts
let countVarible
function getDataFunction
//  ... etc
```
same for components, it can be just Header, Footer, About
files where u store this components have a .component.tsx postfix so other 'potential developer in a big project' will understand that it is a component
So suggest u to rename your entities